packages/scaffolding-cli/templates/src/csr/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
Remove

Reply:
So when I do the eslint story this will be tidied up.